mz_os.h

```diff
@@ -25,6 +25,8 @@ extern "C" {
 #  define MZ_VERSION_MADEBY_HOST_SYSTEM (MZ_HOST_SYSTEM_UNIX)
 #elif defined(_WIN32)
 #  define MZ_VERSION_MADEBY_HOST_SYSTEM (MZ_HOST_SYSTEM_WINDOWS_NTFS)
+#else
```

Comment:
No need for `#elif defined(__unix__) above. We will assume all non-mentioned/non-detectible OSes as UNIX with our new `#else`.

Reply:
True.